playbooks/common/openshift-node/config.yml

```diff
@@ -1,25 +1,4 @@
 ---
-- name: Gather and set facts for node hosts
-  hosts: oo_nodes_to_config
-  vars:
-    t_oo_option_node_debug_level: "{{ lookup('oo_option', 'openshift_node_debug_level') }}"
-  pre_tasks:
-  - set_fact:
-      openshift_node_debug_level: "{{ t_oo_option_node_debug_level }}"
-    when: openshift_node_debug_level is not defined and t_oo_option_node_debug_level != ""
-  roles:
-  - openshift_facts
-  tasks:
-  # Since the master is generating the node certificates before they are
-  # configured, we need to make sure to set the node properties beforehand if
-  # we do not want the defaults
```

Comment:
First time I was reading this comment I was not sure if we can set all the facts at one place. Reading it for the second time and checking the plays between this play up to plays running the `openshift_node` role I can't see anything touching the node certificates. So I believe the comment is say that when running the `openshift-node/config.yml` playbook, the master has already generated the node certificates and we should update the node facts for that reason. If we do it right here or if we do it once we run the `openshift_node` role, the effect will be the same. Prove me otherwise.